src/ProophessorDo/Model/User/Exception/InvalidEmailAddress.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/*
+ * This file is part of prooph/proophessor.
+ * (c) 2014-2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 5/2/15 - 6:56 PM
+ */
+namespace Prooph\ProophessorDo\Model\User\Exception;
+
+/**
+ * Class InvalidName
+ *
+ * @package Prooph\ProophessorDo\Model\User\Exception
+ * @author Alexander Miertsch <kontakt@codeliner.ws>
+ */
+final class InvalidEmailAddress extends \InvalidArgumentException
+{
```

Comment:
Not out of date. My problem was that the [valueobjects](https://github.com/nicolopignatelli/valueobjects) dependency was missing so email validation did not work. But adding the dependency caused version conflicts with other symfony deps. So I decided to remove valueobjects and use native php functions to validate an email. The exception is required now because php's filter_var function does not throw an exception but just returns false.